api_tests/src/shared.ts

```diff
@@ -863,6 +863,7 @@ export function getCommentParentId(comment: Comment): number | undefined {
   if (split.length > 1) {
     return Number(split[split.length - 2]);
   } else {
+    console.log(`Failed to extract comment parent id from ${comment.path}`);
     return undefined;
```

Comment:
There are regular CI failures from this code, I want to figure out whats the reason.

https://woodpecker.join-lemmy.org/repos/129/pipeline/4397/21#L362

Reply:
[Here](https://woodpecker.join-lemmy.org/repos/129/pipeline/4410/21#L350) is one such failure (the previous one in line 347 is expected as part of the test. Im not familiar with the way comment paths work, but a path of `0` should probably never exist, right?